hydra-configs-torch/hydra_configs/torch/optim/__init__.py

```diff
@@ -14,6 +15,25 @@
 from .lbfgs import LBFGSConf
 from . import lr_scheduler
 
+from hydra.core.config_store import ConfigStore
+
+
+def register_configs():
+    cs = ConfigStore.instance()
+    cs.store(provider="torch", group="torch/optim", name="adadelta", node=AdadeltaConf)
+    cs.store(provider="torch", group="torch/optim", name="adagrad", node=AdagradConf)
+    cs.store(provider="torch", group="torch/optim", name="adam", node=AdamConf)
+    cs.store(provider="torch", group="torch/optim", name="adamw", node=AdamWConf)
+    cs.store(
+        provider="torch", group="torch/optim", name="sparseadam", node=SparseAdamConf
+    )
+    cs.store(provider="torch", group="torch/optim", name="adamax", node=AdamaxConf)
+    cs.store(provider="torch", group="torch/optim", name="asgd", node=ASGDConf)
+    cs.store(provider="torch", group="torch/optim", name="sgd", node=SGDConf)
+    cs.store(provider="torch", group="torch/optim", name="lbfgs", node=LBFGSConf)
+    cs.store(provider="torch", group="torch/optim", name="rprop", node=RpropConf)
+    cs.store(provider="torch", group="torch/optim", name="rmsprop", node=RMSpropConf)
```

Comment:
You can use the undocumented ConfigStoreWithProvider (defined next to ConfigStore):
```python
        with ConfigStoreWithProvider("torch") as cs:
          cs.store(group="torch/optim", name="adamax", node=AdamaxConf)
          cs.store(group="torch/optim", name="asgd", node=ASGDConf)
          ...
```

Reply:
you missed this comment?

Reply:
I saw it, just forgot to commit the change I made locally